src/Microsoft.DotNet.Build.Tasks/Targets/PerfTesting.targets

```diff
@@ -4,8 +4,6 @@
   <PropertyGroup>
     <RunPerfTestsForProject Condition="'$(Performance)' == 'true' and '$(RunPerfTestsForProject)' != 'false' and '$(OS)' == 'Windows_NT' and Exists('$(MSBuildProgramFiles32)\MSBuild\Microsoft\Portable\v5.0\Microsoft.Portable.CSharp.targets')" >true</RunPerfTestsForProject>
```

Comment:
orthogonal but can you please remove this check "Exists('$(MSBuildProgramFiles32)\MSBuild\Microsoft\Portable\v5.0\Microsoft.Portable.CSharp.targets')" after my change https://github.com/dotnet/buildtools/pull/294 I don't expect this to be required any longer.